tests/test_camera.py

```diff
@@ -1,76 +0,0 @@
-import os
```

Comment:
Did these camera tests still pass? If so i'd suggest leaving them in, unless you want to remove _all_ the camera api with it